packages/lit-auth-client/src/lib/lit-auth-client.ts

```diff
@@ -146,7 +156,9 @@ export class LitAuthClient {
 
     // Set RPC URL
     this.rpcUrl =
-      options?.rpcUrl || this.litNodeClient.config.litNetwork === 'datil-dev'
+      options?.rpcUrl ||
+      this.litNodeClient.config.litNetwork === 'datil-dev' ||
+      this.litNodeClient.config.litNetwork === 'datil-test'
```

Comment:
If `options.rpcUrl` has a value shouldn't `this.rpcUrl` utltimately be that value?

![Screenshot from 2024-07-11 13-34-49](https://github.com/LIT-Protocol/js-sdk/assets/10857348/7a1c2219-7225-4270-8aaa-de3c36c783c4)

Same above [here](https://github.com/LIT-Protocol/js-sdk/pull/523/files#diff-2dcc2c94e7788b1bbaef5e6cf9158025a2580d238f807bd3cb923bbe355eb47bR619-R621)

Reply:
I hope prettify doesn't do that! The code is fine, just that the operator resolution order does not match the developer intent

Anyway, the other PR is the way to go IMO so marking this as solved and approving 🚀